creator-node/src/middlewares.js

```diff
@@ -266,18 +266,29 @@ async function ensureStorageMiddleware(req, res, next) {
 /**
  * Issue SyncRequests to both secondaries, and wait for at least one to sync before returning
  * @dev TODO - move out of middlewares layer
+ * @param ignoreWriteQuorum true if write quorum should not be enforced (don't fail the request if write quorum fails)
  */
-async function issueAndWaitForSecondarySyncRequests(req) {
+async function issueAndWaitForSecondarySyncRequests(
+  req,
+  ignoreWriteQuorum = false
+) {
   const serviceRegistry = req.app.get('serviceRegistry')
   const { snapbackSM } = serviceRegistry
 
   // Parse request headers
   const pollingDurationMs =
     req.header('Polling-Duration-ms') ||
     config.get('issueAndWaitForSecondarySyncRequestsPollingDurationMs')
+  // Write quorum header always takes precedence over env var if explicitly defined
+  // Empty/undefined header means enforceWriteQuorum decides if write quorum is enabled
+  const writeQuorumHeaderTrue = !!req.header('Enforce-Write-Quorum')
```

Comment:
Just curious if the plan to have the client pass this header in as a feature flag?

Reply:
yeah I think a feature flag makes the most sense for rollout like Dheeraj was saying above. I asked about the timeline of that in the discussion thread above as well

Reply:
@raymondjacobson @theoilie yes that's the thinking. basically we can turn it on/off if upload rates degrade vs hardcoding as a server side env var which would take a release to flip. We actually have an optimizely flag but i don't think it ever got hooked up on the client/libs side

Reply:
hmm I didn't see any flags when I searched "write" or "quorum" in optimizely but just made one this morning. I think I should prioritize the libs/client side of this first since that blocks rolling it out

Reply:
lmk if you have Qs on how to do that @theoilie

Reply:
@theoilie here's the link https://app.optimizely.com/v2/projects/18140642080/features/21790311781/rules

Reply:
@theoilie MVP